src/utils.jl

```diff
@@ -1,6 +1,8 @@
 #
 """ use Base.ReshapedArray """
 _reshape(a, dims::NTuple{D,Int}) where{D} = reshape(a,dims)
+_reshape(a::ReshapedArray, dims::NTuple{D,Int}) where{D} = _reshape(a.parent, dims)
```

Comment:
This is piracy

Reply:
no because `_reshape` is a method defined in `SciMLOperators`, and is not exported either

per https://docs.julialang.org/en/v1/manual/style-guide/#Avoid-type-piracy, ""Type piracy" refers to the practice of extending or redefining methods in Base or other packages on types that you have not defined."

`_reshape` is used to get around https://github.com/JuliaLang/julia/issues/36313.  There's reasonable improvement in https://github.com/JuliaLang/julia/pull/43724 and https://discourse.julialang.org/t/convert-array-into-matrix-in-place/55624/3 with this change.